src/simularium/RemoteSimulator.ts

```diff
@@ -412,4 +446,19 @@ export class RemoteSimulator implements ISimulator {
             "Convert trajectory output to simularium file format"
         );
     }
+
+    public checkServerHealth(): Promise<void> {
+        return this.connectToRemoteServer()
+            .then(() => {
+                this.webSocketClient.sendWebSocketRequest(
+                    {
+                        msgType: NetMessageEnum.ID_CHECK_HEALTH_REQUEST,
+                    },
+                    "Request server health check"
+                );
+            })
+            .catch((e) => {
+                throw new FrontEndError(e.message, ErrorLevel.ERROR);
+            });
```

Comment:
Questions about error handling.

UX request is that if "server is down" we throw a styled modal.

But if the viewer can't connect this error also generates up a full screen React runtime error.

Removing this error handling yields the behavior UX asked for (just a modal), but is that not proper error handling?